app/models/imageable.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: imageables
+#
+#  id             :bigint           not null, primary key
+#  imageable_type :string           not null
+#  picture        :string
+#  type           :integer
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  imageable_id   :bigint           not null
+#
+# Indexes
+#
+#  index_imageables_on_imageable  (imageable_type,imageable_id)
+#
+class Imageable < ApplicationRecord
+  validates :image, presence: true, uniqueness: true
+  enum type: %i[type1 type2 type3]
```

Comment:
Maybe for now we set two types? Icons and Photo

Reply:
Alright!